tests/data/loader/sxp/config.yaml

```diff
@@ -0,0 +1,97 @@
+core:
+  loader: sxp
+  paths:
+    data_raw_dir: "tests/data/loader/sxp/"
+    data_parquet_dir: "tests/data/loader/sxp/parquet"
+
+binning:
+  num_cores: 10
+
+dataframe:
+  ubid_offset: 0
+  daq: DA03
+  forward_fill_iterations: 2
+  x_column: dldPosX
+  corrected_x_column: "X"
+  kx_column: "kx"
+  y_column: dldPosY
+  corrected_y_column: "Y"
+  ky_column: "ky"
+  tof_column: dldTimeSteps
+  tof_ns_column: dldTime
+  corrected_tof_column: "tm"
+  bias_column: "sampleBias"
+  tof_binwidth: 2.0576131995767355E-11 # in seconds
+  tof_binning: 3
+  jitter_cols: ["dldPosX", "dldPosY", "dldTimeSteps"]
+
+  units:
+    dldPosX: 'step'
+    dldPosY: 'step'
+    dldTimeSteps: 'step'
+    tof_voltage: 'V'
+    extractorVoltage: 'V'
+    extractorCurrent: 'A'
+    cryoTemperature: 'K'
+    sampleTemperature: 'K'
+    dldTime: 'ns'
+    # delay: 'ps'
+    timeStamp: 's'
+    # energy: 'eV'
+    # E: 'eV'
+    kx: '1/A'
+    ky: '1/A'
+
+  channels:
+    timeStamp:
+      format: per_train
+      dataset_key: "/INDEX/timestamp"
```

Comment:
Not for this PR necessarily, but coming up with a schema for the dataframe in config file would be great. 
I think we had the discussion to use the group_names in mpes too.

Reply:
This kind of defines a schema, but yes, reworking the mpes structure here is on my todo (not very high up though).